fiftyone/operators/executor.py

```diff
@@ -469,6 +503,21 @@ def to_dict(self):
             k: v for k, v in self.__dict__.items() if not k.startswith("_")
         }
 
+    def set_progress(self, progress: float = None, label: str = None):
+        """Sets the progress of the current operation.
+
+        Args:
+            label: string label to display
+            progress: the progress to set
```

Comment:
Maybe use the same doc string from `ExecutionProgress`?
```suggestion
        progress (None): an optional ``float`` between 0 and 1 (0% to 100%)
        label (None): an optional label to display
```